### ISSUE TITLE
Arduino Clone with CH340 wihtout FTDI driver

### ISSUE THREAD

**Issue by Phoenix-64** [projectgus/hairless-midiserial#68]
Can I youse Hairless Midi with the Ch340 chip or does it just simply not work?

**Comment by vifyrv** [comment]
Quite a ways late to reply. But it looks like it’s been six months or so and nobody has replied.

I too have been having a similar issue but with windows. I’ve gotten Hairless midi to work but with little success. I.e. it works for about 1 minute and then stops. Look up FTDIgate and that might let you in on what is happening. I’m finding my most logical answer to switch to a Teensy 4.0 or a more official arduino nano that has the FTDI chip instead of CH340. Even though I think it’s unethical what FTDI has done.

Did you ever find a solution to your problem on mac or did you just switch to a different board?

**Comment by Phoenix-64** [comment]
No, unfortunately not. I just completely ditched the MIDI part and created a python script that read the serial output of the Arduino
